src/styles/sections/footer.scss

```diff
@@ -1,5 +1,13 @@
 .footer {
-  background: #202D33;
+  background: #202d33;
   text-align: center;
   padding: 30px;
-}
\ No newline at end of file
+}
+
+.cookies-note {
+  color: white;
```

Comment:
```suggestion
  color: #ffffff;
```

Navrhoval bych definovat barvy v HEX číslech ať je to konzistentní. Vídím, že tam máme několik variant bíle barvy -`#FFFFFF`, `#fff` a `#ffffff`, co to třeba sjednotit a dát jednoho formátu? :)

Reply:
Pardon, to mi uniklo. Běžně tu slovní variantu necommituju. Mám teda vytvořit nový pull request s opravou všech stylů? Formát #FFFFFF? Pro všechy barvy?

Reply:
Ja bych to asi klidně udělal v tomhle PR, možná @HormCodes má jiný názor?

Reply:
Za mne separé commit v tomto PR je OK.

Reply:
Mimochodem good point. Můžeme to rovnou přidat do konvencí. 👍

Reply:
LGTM, merguji a díky @poooow 👍